x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/latest/LatestConfigTests.java

```diff
@@ -62,59 +64,47 @@ public void testValidate_ValidConfig() throws IOException {
     }
 
     public void testValidate_EmptyUniqueKey() throws IOException {
-        String json = "{"
-            + " \"unique_key\": [],"
-            + " \"sort\": \"timestamp\""
-            + "}";
+        String json = "{" + " \"unique_key\": []," + " \"sort\": \"timestamp\"" + "}";
```

Comment:
Do you plan to apply
```
// tag::NO_CODE_FORMAT
```
here and in other test cases?
Now it looks strange IMO to have this concatenations in one line.

Reply:
sorry, I did not fix those, will do.